subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -611,7 +610,7 @@ static int cmd_wifi_connect(const struct shell *sh, size_t argc,
 			    char *argv[])
 {
 	struct net_if *iface = net_if_get_wifi_sta();
-	struct wifi_connect_req_params cnx_params = { 0 };
+	struct wifi_connect_req_params cnx_params = {0};
```

Comment:
I thought that the rule was opposite i.e., to have spaces there, at least it was like that in the past, but perhaps this has changed.

Reply:
FWIW after dumping with following command since Zephyr use `BasedOnStyle:  LLVM`

`clang-format -style=llvm -dump-config > .clang-format-llvm`

Seems this configuration was enabled `Cpp11BracedListStyle: true`
https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
There are many styles possible, not sure this one is the best for us.

Could you take this commit away from this PR, lets figure out the used style later.

Reply:
Yes, let's follow existing style in Zephyr, if needed a separate PR that fixes style in enitre repo can be raised.

Reply:
@jukkar Rather than take this commit away, how about to remove just the following?
- Reformatted `{ 0 }` to `{0}` as per .clang-format `settings` for uniformity.

Reply:
Sure, that is fine